modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/DeliverableAndesMetadata.java

```diff
@@ -251,10 +251,14 @@ public void markAsDispatchedToDeliver(UUID channelID) {
         channelInformation.addChannelStatus(ChannelMessageStatus.DISPATCHED);
 
         if (!this.isBeyondLastRollbackedMessage) {
-            channelInformation.incrementDeliveryCount();
+            int i = channelInformation.incrementDeliveryCount();
+            MessageTracer.trace(getMessageID(), getDestination(), "Message within rollback id. "
+                    + "Increased count " + i);
```

Comment:
I suppose increased count is a bit misleading. We can use the term current count as used in the else block

Reply:
Fixed.